packages/ds-component/__tests__/impl.test.ts

```diff
@@ -669,6 +669,22 @@ test('objectTransform works', () => {
   expect(actual).toEqual(expected);
 });
 
+test('identity transform logs warning', () => {
+  const message = testMessage(1, 1, 1);
+  global.console.warn = jest.fn((warn) => {
+    throw new Error(warn);
```

Comment:
Prefer using `mock.calls[0][0]` to test that this was called with the right value instead of making this throw. 

See [Using a mock function]

[Using a mock function]: https://jestjs.io/docs/en/mock-functions.html#using-a-mock-function

Reply:
I'm somehow not able to get it working without the throw, but I added a mock.calls to check the value it's called with.